### ISSUE TITLE
Glob support for `sourcemaps` 

### ISSUE THREAD

**Issue by austince** [getsentry/action-release#116]
It would be nice not to have to expand this ourselves, eg. just:

```yaml
with:
  sourcemaps: ui/build/static/js/*.map
````

This could probably be done with just running the array through [minimatch](https://www.npmjs.com/package/minimatch) or such. 

_Originally posted by @amcdnl in https://github.com/getsentry/action-release/issues/43#issuecomment-766800395_

**Comment by armenzg** [comment]
Thanks @austince for filing! At the moment, we're not currently investing in the action but we will look at this when we do.

**Comment by armenzg** [comment]
> Nice, thanks @armenzg!
> 
> > we're not currently investing in the action
> 
> Does this apply to any changes, or just writing the feature? We may be able to contribute a PR if you'd have the bandwidth to review, but understand if not.

We welcome PRs! Tag me and I will be able to help.

**Comment by andreiborza** [comment]
Yes, it goes through all subdirectories and finds source maps to upload. We could think of adding glob support but it's not a high priority at the moment. Contributions are welcome tho.
